<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SdCardMemoryLack</class>
 <widget class="QWidget" name="SdCardMemoryLack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="CustomLabel" name="warningLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="CustomLabel" name="warningLabel2">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="CustomLabel" name="sdCardValueLabel">
       <property name="maximumSize">
        <size>
         <width>57</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CustomLabel" name="label_4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CustomLabel" name="label_5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="CustomPushButton" name="pushButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomPushButton</class>
   <extends>QPushButton</extends>
   <header>CustomPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>CustomLabel</class>
   <extends>QLabel</extends>
   <header>CustomLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
